src/main/java/funbox/util/TaskList.java

```diff
@@ -0,0 +1,193 @@
+package funbox.util;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.Locale;
+
+import funbox.exception.FunBoxExceptions;
+import funbox.task.Task;
+import funbox.task.Deadline;
+import funbox.task.Event;
+
+public class TaskList {
+    private ArrayList<Task> taskList;
```

Comment:
Not too sure about this but I think ArrayList is considered under Collections, and the coding standards ask for a plural form on names representing a collection. The naming sounds ok since `taskLists` would sound weird, but maybe consider `tasks`? Although if it causes ambiguity it might be good to check if `taskList` is accepted!